tb_plugin/packaging/torch_tb_profiler/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: torch_tb_profiler
+  version: "{{ environ.get('BUILD_VERSION', '0.2.0') }}"
+
+source:
+ path: "{{ environ.get('SOURCE_ROOT_DIR', '../..') }}"
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pytorch
+
+  run:
+    - python
+    - pandas
+    - tensorboard
+
+build:
+  string: py{{py}}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+test:
+  imports:
+    - torch_tb_profiler
+  source_files:
+    - test
+  requires:
+    - tensorboard
+    - pandas
```

Comment:
Why do we need pandas?

Reply:
Please see the install dependencies for torch-tb-profiler [here](https://github.com/pytorch/kineto/blob/main/tb_plugin/ci_scripts/install_env.sh#L23)

Reply:
This is not the source of truth for packaging system, but this is:
https://github.com/pytorch/kineto/blob/eaaf2acf8811e93f0175a904e63ee864cc884162/tb_plugin/setup.py#L27-L30

And the wheel that is currently published also depends on pandas:
```
% curl -O https://files.pythonhosted.org/packages/ac/53/49685c0b0d2388959fa58a1b6827bf7c565c91e01862aadde42c1ac10ad8/torch_tb_profiler-0.2.1-py3-none-any.whl; python3 -c 'import pkginfo;print(pkginfo.get_metadata("torch_tb_profiler-0.2.1-py3-none-any.whl").requires_dist)'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  932k  100  932k    0     0  5585k      0 --:--:-- --:--:-- --:--:-- 5585k
['pandas (>=1.0.0)', 'tensorboard (!=2.1.0,>=1.15)', "azure-storage-blob ; extra == 'blob'", "google-cloud-storage ; extra == 'gs'", "boto3 ; extra == 's3'"]
```